### PR TITLE
Add Python 3.6 on Linux

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,16 +13,14 @@ source:
...
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 
 requirements:
   build:
     - python
     - setuptools
-    - botocore >=1.3.0,<2.0.0
-    - futures >=2.2.0,<4.0.0  # [not py3k]
```
Comment (hajapy) [inline recipe/meta.yaml]: These requirements are specified by s3transfer's setup.py
https://github.com/boto/s3transfer/blob/0.1.10/setup.py#L14
https://github.com/boto/s3transfer/blob/0.1.10/setup.py#L21

Are you sure they can be removed?

Comment (jakirkham) [inline recipe/meta.yaml]: Only removing them from build time. They are still required at install time.

Comment (hajapy) [inline recipe/meta.yaml]: Oh I see, sounds good.

**Comment by jakirkham** [comment]
Appears to be green. 💚 Anything else?

**Comment by hajapy** [comment]
Looks good, though since I am not a maintainer for this recipe (just an interested party :P) I cannot merge it for you.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**